Dockerfile

```
@@ -1,12 +1,12 @@
-FROM codait/max-base:v1.1.0
+FROM codait/max-base:v1.1.1
 
 ARG model_bucket=http://max-assets.s3-api.us-geo.objectstorage.softlayer.net/audio-embedding-generator/1.0
```

Can you please update the cos link while you're at it?
```suggestion
ARG model_bucket=http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/audio-embedding-generator/1.0
```